assets/course-theme/blocks/lesson-blocks/course-title/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import courseTitleMeta from './block.json';
+import { CourseTitleEdit } from './course-title-edit';
+
+import LogoTreeIcon from '../../../../icons/logo-tree.svg';
+
+export default {
+	...courseTitleMeta,
```

Comment:
Just curious why the other meta isn't being carried over?

Reply:
That meta was not necessary because "category" was already defined https://github.com/Automattic/sensei/pull/6762/files#diff-a8e00844be227e06b70f227b0c8fc48ffbef33b3376eddbb483dfe763a6b6703R3